zappa/zappa.py

```diff
@@ -326,9 +334,19 @@ def create_lambda_zip(self, prefix='lambda_package', handler_file=None,
                 venv = bin_path[:bin_path.rfind(env_name)] + env_name
                 logger.debug('env path = {}'.format(venv))
             else:  # pragma: no cover
-                print("Zappa requires an active virtual environment.")
+                print("Zappa requires an active virtual or conda environment.")
                 quit()
 
+            if conda_mode:
```

Comment:
Except for the precompiled_packages warning, why does this matter if its conda_mode or not?